docs/pilot_documentation/general_pilot.rst

```diff
@@ -18,6 +18,8 @@ It also implements some general logic used by all pilots.
 This ensures a consistent interface for all pilots and allows for easy extension of the pilot framework.
 For more information on how to create a new Pilot, refer to this Tutorial:
 
+Supported/Tested Gates on IBM and AWS: X, Y, Z, H, CX, CXX, S, T
```

Comment:
Bezüglich unterschiedlicher Bennenung könnte man das auch zu den Verfügbaren Sprachen schieben und dort definieren. (Also was in QASM2, QASM3 usw. möglich ist).